### DISCUSSION TITLE
Wildcard cert – DNS Challenge via  ACME-DNS

### DISCUSSION THREAD

**Discussion by msteinhartOPC** [NginxProxyManager/nginx-proxy-manager#1876] [Q&A]
I have been using Nginx Proxy Manager for over a year on multiple networks and I find it the easiest and most reliable way to configure reverse proxies with Lets Encrypt certs. On a few subdomains (local networks) I need wildcard cert for several systems. Some of the networks are single public IP with Nginx Proxy Manager utilizing ports 80 and 443. It looks like the best way on these networks is to have Nginx Proxy Manager  (request, manage and auto-update) the certs for the subdomains.
I see that ACME-DNS is one of the providers listed in the DNS Provider list but no documentation.
The question is how to use Nginx Proxy Manager with ACME-DNS.

Is ACME-DNS integrated into the docker container of Nginx Proxy Manager?
Is Nginx Proxy Manager calling an external ACME-DNS?
How to use NPM with ACME-DNS?

Thanks.

**Comment by Kelsch** [comment]
Did you ever figure out how to use this?

**Comment by Seg0815** [comment]
Hi, I am looking for a solution like this an found this [video](https://www.youtube.com/watch?v=7jEzioFsyNo&t=923s).
I will install the acme-dns service on my public part of my network and hope to configure it and nginx-proxy-manager in my private network correct so the dns challenges are functional.

**Comment by flypenguin** [comment]
this works kinda beautifully for me (example for AWS route53) ...

<img width="549" height="735" alt="29291" src="https://github.com/user-attachments/assets/6082bf41-bfb3-4831-b621-8d8b066edbb9" />

**Reply by Seg0815** [reply]
Of course a suitable solution for your setup, i guess.
My name service provider does not support dns challenges - there is no possibility to use a plugin out of the box.
I have allready installed my own name service (acme-dns) for a sub domain (according to the posted video and other post) and configured the reverse proxy with the acme-dns plugin. It worked, I think.

**Comment by flypenguin** [comment]
@Seg0815 I am not sure if you're getting it wrong or simply used non-precise wording. _every_ provider does support DNS01 challenges, it's simply setting a DNS entry – there is absolutely no provider-specific functionality required. You could even do it by hand.

Probably you meant that your provider is not among the ones supported by your particular client, which, yes, happens to the best of them.

Sorry if you already meant that, then maybe someone else comes by this ticket using the same search as I did and finds this information useful. (I had that situation often where someone commented "solved it" and that's it – on a problem I had myself, and I did _not_ solve it yet ... makes me furious of the universe and all creation in it every time 😉)

**Reply by Seg0815** [reply]
Please forgive my inaccuracy: in fact, my provider does not offer an API for automating the DNS challenge.
